ppa/archive/solr.py

```diff
@@ -91,6 +91,27 @@ class SolrSchema(object):
                 ]
             },
         },
+        {
+            'name': 'punctuation_parser',
```

Comment:
"parser" doesn't make sense to me as a field name, maybe something like `text_punctuation` or `text_with_punctuation`? Strictly speaking, this isn't specific to punctuation per se, that's just one of our use cases. What about something like `text_simpletoken`